packages/build/src/error/parse/parse.js

```diff
@@ -40,12 +40,12 @@ const getFullErrorInfo = function ({ error, colors, debug }) {
 // Parse error instance into all the basic properties containing information
 const parseErrorInfo = function (error) {
   const { message, stack, ...errorProps } = normalizeError(error)
-  const errorInfo = getErrorInfo(errorProps)
+  const [errorInfo, errorPropsA] = getErrorInfo(errorProps)
```

Comment:
Ensures we remove the `errorInfo` property from the error static properties printed in build logs.